lib/codecs/src/decoding/format/native_json.rs

```diff
@@ -46,6 +46,8 @@ impl Deserializer for NativeJsonDeserializer {
     fn parse(
         &self,
         bytes: Bytes,
+        // LogNamespace is ignored because Vector owns the data format being consumed and as such there
+        // is no need to change the fields of the event.
```

Comment:
Copy the comment to `native.rs` as well?

Reply:
oh wth.. I had it in both places. I bet didn't write the file. blaming adjusting to helix :sweat_smile:

Reply:
yup. forgot to write.